modules/gcp/oslogin.go

```diff
@@ -0,0 +1,90 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/oslogin/v1"
+)
+
+// Add an OS Login SSH Key
+func ImportSSHKey(t *testing.T, user, key string) {
+	err := ImportSSHKeyE(t, user, key)
+	if err != nil {
+		t.Fatalf("Could not add SSH Key to user %s: %s", user, err)
+	}
```

Comment:
We have a new pattern now that uses require, so this should be:

```
require.NoErrorf(t, ImportSSHKeyE(t, user, key), "Could not add SSH Key to user %s: %s", user, err)
```

Reply:
Done- not sure how to use this in cases with returns like https://github.com/gruntwork-io/terratest/pull/267/files#diff-672950984cb105f6fbcfa972d0119285R54

Reply:
You can do `require.NoErrorf(t, err, ...)`. It's not a big deal, but some of us have been finding it more readable when we drop the indent.